src/editors/containers/ProblemEditor/data/ReactStateOLXParser.js

```diff
@@ -7,15 +7,16 @@ class ReactStateOLXParser {
     const parserOptions = {
       ignoreAttributes: false,
       alwaysCreateTextNode: true,
-      // preserveOrder: true
     };
     const builderOptions = {
       ignoreAttributes: false,
       attributeNamePrefix: '@_',
       suppressBooleanAttributes: false,
       format: true,
     };
+    this.parserOptions = parserOptions;
```

Comment:
Instead of declaring `parserOptions` seperately, we can declare here directly
```suggestion
    this.parserOptions = {
      ignoreAttributes: false,
      alwaysCreateTextNode: true,
    };
```